readme.md

```diff
@@ -169,6 +167,37 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+## Reporters
+
+### Mini-reporter
+
+![](media/screenshot-mini-reporter.gif)
+
+### Verbose reporter
+
+The verbose reporter is always used in CI environments unless [`--tap`](#optional-tap-output) is specified.
+
+<img src="media/screenshot.png" width="150">
+
+### Optional TAP output
```

Comment:
All caps (`TAP`) is the preferred representation based on the website http://testanything.org